### PR TITLE
bigz/add-entry-mining-script

### PR DESCRIPTION
**Pull request by 0xbigz**

### CODE REVIEW

**Comment by evanpipta** [comment]
I don't have too much of the context behind this, but some random thoughts:
- Seems like it should be added to the scripts section of package.json?
- Should there be a readme explaining lightly how to run this, if we expect it to be used by non-devs?

File: scripts/mineAdditionalEntries.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+		for (let i = 1; i <= n; i++) {
+			console.log('claiming! entry:', i);
+			try {
+				competitionClient.claimEntry(
+					competitionKey,
+					competitorKey,
+					userStatsKey
+				);
+				await sleep(1000);
```
Comment (evanpipta) [inline scripts/mineAdditionalEntries.ts:71]: Wait, so this will run for 10k seconds if the default value for n is used?

Comment (0xbigz) [reply scripts/mineAdditionalEntries.ts:71]: the default was overriden but updated so its required and default without --param is 5

**Comment by 0xbigz** [comment]
> I don't have too much of the context behind this, but some random thoughts:
> 
> * Seems like it should be added to the scripts section of package.json?
> * Should there be a readme explaining lightly how to run this, if we expect it to be used by non-devs?

added! good points

**Pull request merged**